### DISCUSSION TITLE
Bar chart instead of line chart

### DISCUSSION THREAD

**Discussion by oscarbaruffa** [plausible/analytics#1866] [Anything Else]
The default chart type for all charts are lines, but a bar chart is usually better for showing binned data. You could default the bins to days/months/years depending on the period selection

**Comment by metmarkosaric** [comment]
thanks @oscarbaruffa! we'll be finalizing the work on allowing you to change intervals on the chart pretty soon (see https://github.com/plausible/analytics/pull/1574) and perhaps looking at how we can improve the chart display will be something to look at after that. no promises at this stage as we have many other popular requests at the moment
